pom.xml

```diff
@@ -378,7 +295,8 @@
     <!-- Work around dataflow, grpc, and protobuf version issues
          http://stackoverflow.com/questions/36257708/java-lang-noclassdeffounderror-while-deserializing-protobuf-messages -->
     <google.dataflow.version>1.6.0</google.dataflow.version>
-    <io.grpc.version>0.14.0</io.grpc.version>
+    <io.netty.version>4.1.0.Final</io.netty.version>
```

Comment:
Shouldn't this be `4.1.1.Final` instead based on the following:

```XML
<dependency>
  <groupId>io.netty</groupId>
  <artifactId>netty-codec-http2</artifactId>
  <version>[4.1.1.Final]</version>
  <scope>compile</scope>
</dependency>
```
The above is from the following pom in Maven:

https://repo1.maven.org/maven2/io/grpc/grpc-netty/0.15.0/grpc-netty-0.15.0.pom